projects/habitat_ovmm/eval_vectorized.py

```diff
@@ -251,7 +249,7 @@ def stop():
     config, config_str = get_config(args.habitat_config_path, opts=args.opts)
     OmegaConf.set_readonly(config, True)
     baseline_config = OmegaConf.load(
-        "projects/habitat_ovmm/configs/agent/floorplanner_eval.yaml"
+        "projects/habitat_ovmm/configs/agent/floorplanner_detic_eval.yaml"
```

Comment:
We should add a command line flag for this?

Reply:
There already exists one. Using it now. Updated command in README